files/fr/web/css/@font-face/size-adjust/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: size-adjust
+slug: Web/CSS/@font-face/size-adjust
+browser-compat: css.at-rules.font-face.size-adjust
+translation_of: 'Web/CSS/@font-face/size-adjust'
+---
+<p>{{CSSRef}}{{SeeCompatTable}}</p>
+
+<p>Le descripteur CSS <strong><code>size-adjust</code></strong> CSS définit un multiplicateur destiné aux contours des glyphes et au mesures associées à la police. Cela facilite l'harmonisation de l'apparence des polices lorsqu'elles sont rendues avec la même taille.</p>
```

Comment:
minor typo "au mesures" > "aux mesures"

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1755/commits/f24626b749ca42e25dad1fdf7c0a02f83b965931